### DISCUSSION TITLE
The `include_all_commits` parameter is not working.

### DISCUSSION THREAD

**Discussion by bavo96** [anuraghazra/github-readme-stats#3303] [Q&A] [answered]
Hi guys,

Recently, the API using `include_all_commits` is not working anymore:

With `include_all_commits`: https://github-readme-stats.vercel.app/api?username=bavo96&theme=gruvbox&show_icons=true&hide=contribs&include_all_commits=true
![image](https://github.com/anuraghazra/github-readme-stats/assets/27908949/c64d1bd8-dd63-4566-83c3-99767ccb4225)
Without `include_all_commits`: https://github-readme-stats.vercel.app/api?username=bavo96&theme=gruvbox&show_icons=true&hide=contribs&include_all_commits=false
![image](https://github.com/anuraghazra/github-readme-stats/assets/27908949/89a86ea6-70a3-459a-ae7d-4069b49e1119)

Can anyone help me with this?

**Comment by qwerty541** [comment, marked as answer]
Hey, @bavo96! You can see this when upstream GitHub API respond with error due to some reason during fetching stats card data. It should be resolved automatically after some time. Recently we merged https://github.com/anuraghazra/github-readme-stats/pull/3238 which improved confusing behavior of showing 0 commits, currently we show error card instead.

**Reply by bavo96** [reply]
Many thanks for your information, the upstream seems to be fixed. I've seen my GitHub stats again :D
